pandas/io/tests/test_packers.py

```diff
@@ -543,25 +543,6 @@ def test_sparse_frame(self):
         self._check_roundtrip(ss3, tm.assert_frame_equal,
                               check_frame_type=True)
 
-    def test_sparse_panel(self):
-
```

Comment:
I don't really follow.  How are we supposed to test pickling `sparse_panel` if it doesn't exist.